src/openqaoa-core/algorithms/rqaoa/rqaoa_utils.py

```diff
@@ -641,47 +619,33 @@ def solution_for_vanishing_instances(hamiltonian: Hamiltonian, spin_map: dict):
     Returns
     -------
     cl_energy: `float`
-        The energy of the first solution wrt the cost Hamiltonian.
+        The energy of the solution wrt the cost Hamiltonian.
     cl_ground_states: `list`
```

Comment:
I guess the type should be `str`

Reply:
The energy is float, the ground states are a list of strings usually. Here since it's only one, it's a list of a string.

Reply:
Oh, you mean that the return is `cl_ground_state_list` and not `cl_ground_states`?

As it stands it is all very confusing

Reply:
yeah, true, changed, thanks